tmp.go

```diff
@@ -0,0 +1,9 @@
+package moira
+
+type tmp struct {
+	field1 bool
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`field1` is unused (structcheck)